tests/unit/lib/test_file.py

```diff
@@ -323,3 +326,52 @@ def test_read_text(tmp_path, catalog):
     file = File(path=file_name, source=f"file://{tmp_path}")
     file._set_stream(catalog, True)
     assert file.read_text() == data
+
+
+@pytest.mark.parametrize("cloud_type", ["file", "s3", "gs", "azure"], indirect=True)
+def test_resolve_file(cloud_test_catalog):
+    ctc = cloud_test_catalog
+
+    dc = DataChain.from_storage(ctc.src_uri, catalog=ctc.catalog)
+
+    resolved_files = dc.map(file=resolve).collect("file")
```

Comment:
```suggestion
    resolved_files = dc.map(file2=resolve).collect("file2")
```
There seems to be a bug in `.map()` where it doesn't behave correctly if you give it an existing column.